source/index.txt

```diff
@@ -95,11 +100,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Resources
+---------
+
+Visit the Developer Hub and MongoDB University for resources related to
+the MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub contains content, resources, community, and social
+engagement for developers.
+
+To view articles on Java, see the `How To's and Articles page
```

Comment:
I think this info here and a few lines down (describing forums) could be a little more specific/precise. Pretend the user has to pay for every click on the page and therefore needs as much info as possible before deciding to visit a link. It might help to try to answer these questions:

1) Would you characterize the content of the articles as Java or Java Driver? 
2) What kind of articles are these?

Applies to the forums description as well.